Orion Trail: Fix Instructions screen

esainane opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

Before:
![tg-orion-trail-before](https://user-images.githubusercontent.com/14355175/138157375-273b688b-7b3b-45fd-a257-0a1b46e7aaac.png)

After:
![tg-orion-trail-after](https://user-images.githubusercontent.com/14355175/138157388-79eb222e-c5d9-46af-a214-969ebd3c9726.png)


<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game

now these days have gone I'm not so self assured

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog

<!-- If your PR modifies aspects of the game that can be concretely observed by players or admins you should add a changelog. If your change does NOT meet this description, remove this section. -->

:cl:
fix: The Orion Trail's instructions screen is usable again.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

esainane committed:
Orion Trail: Fix Instructions screen

Previously, sections were horribly misshapen, and mostly off past the
right edge of the window.

stylemistake reviewed (changes requested):
If you are removing filling, then there is literally no need for a vertical stack. Either:

- make it work with `fill` prop, or...
- stack Sections directly (without a stack), and add a scrollbar to the window.

esainane committed:
Orion Trail: Pop unnecessary Stack

Also:
 - Add a scrollbar to the Window
 - Slightly increase default window size to fit the new scrollbar
 - Slightly increase default window height to cover typical content

esainane commented:
Thanks for the quick feedback! Popped the Stack, added a scrollbar, and resized to fit.
